docs/source/docker/getting_started/selection.rst

```diff
@@ -0,0 +1,533 @@
+                "type": "DIVERSIFY"
+            }
+
+        If you want to preserve a minimum distance between chosen samples, you 
+        can specify it as an additional stopping condition. The selection process
+        will stop as soon as the stopping criteria has been reached.
+
+        .. code-block:: python
+            :emphasize-lines: 3
+
+            "strategy": {
+                "type": "DIVERSIFY",
+                "stopping_condition_minimum_distance": 0.2
+            }
+
+        Setting :code:`"stopping_condition_minimum_distance": 0.2` will remove all samples which are
+        closer to each other than 0.2. 
+        This allows you to specify the minimum allowed distance between two image
+        embeddings in the output dataset. Since we normalize the input embeddings
+        to unit length, this value should be between 0 and 2.0.
+        This is often a convenient method when working with different data sources and trying to combine them in a balanced way.
+        If you want to use this stopping condition to stop the selection early, make sure that you allow selecting enough samples by setting :code:`n_samples` high enough.
+
+    .. tab:: WEIGHTS
+
+        The objective of this strategy is to **select samples that have a high numerical value**. 
+        It requires the input to be a Nx1 matrix of numbers.
+        
+        Can be used with **SCORES** and **numerical METADATA**. It can be specified easily:
+
+        .. code-block:: python
+
+            "strategy": {
+                "type": "WEIGHTS"
+            }
+
+    .. tab:: THRESHOLD
+
+        The objective of this strategy is to only **select samples that have a numerical value fulfilling a threshold criterion**.
+        E.g. they should be bigger or smaller than a certain value. Like for weighting, this strategy requires the input to be a Nx1 matrix of numbers.
+
+        Can be used with **SCORES** and **numerical METADATA**. It is specified as follows:
+
+        .. code-block:: python
+
+            "strategy": {
+                "type": "THRESHOLD",
+                "threshold": 20,
+                "operation": "BIGGER_EQUAL"
+            }
+
+        The allowed operations are :code:`SMALLER`, :code:`SMALLER_EQUAL`, :code:`BIGGER`, :code:`BIGGER_EQUAL`.
+
+    .. tab:: BALANCE
+
+        The objective of this strategy to **select samples such the distribution of classes in them is as close to a target distribution as possible**.
+
+        E.g. the samples chosen should have 50% sunny and 10% rainy weather.
+        Or, the objects of the samples chosen should be 20% ambulance and 40% buses.
+
+        Can be used with **PREDICTIONS** and **categorical METADATA**.
+
+        .. code-block:: python
+
+            "strategy": {
+                "type": "BALANCE",
+                "target": {
+                    "Ambulance": 0.2, # `Ambulance` should be a valid class in your `schema.json`
+                    "Bus": 0.4
+                }
```

Comment:
Same here, make it add up to 100%. We can always explain later on what happens if it does not add up to 100%.